ord_schema/validations.py

```diff
@@ -446,6 +447,22 @@ def validate_reaction(message: reaction_pb2.Reaction,
 
 def validate_reaction_identifier(message: reaction_pb2.ReactionIdentifier):
     check_type_and_details(message)
+    if Chem and message.type in [
```

Comment:
Under what circumstances will `Chem` be `None`?

Reply:
sorry; just copied from validate_compound_identifier. We made rdkit a formal dep, so I'll remove that.